doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,636 @@
+
+.. code-block:: text
+
+    subsection simulation control
+      set time step        = 5e-5
+      set time end         = 32.5
+      set log frequency    = 500
+      set output frequency = 1200
+      set output path      = ./output_dem/
+    end
+
+Restart
+~~~~~~~~
+
+This simulation reads the restart, meaning this option is set to true. Also, the checkpointing is reduce to 0.5 seconds.
+
+.. code-block:: text
+
+    subsection restart
+      set checkpoint = true
+      set frequency  = 10000
+      set restart    = true
+      set filename   = dem
+    end
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The main difference in this simulation is the gravity. It changes to the y-direction to be coherent with the next simulation using the CFD-DEM solver.
+
+.. code-block:: text
+
+   subsection lagrangian physical properties
+     set g                        = 0, -9.81, 0
+     set number of particle types = 1
+     subsection particle type 0
+       set size distribution type            = uniform
+       set diameter                          = 0.005
+       set number of particles               = 32194
+       set density particles                 = 890
+       set young modulus particles           = 1e6
+       set poisson ratio particles           = 0.33
+       set restitution coefficient particles = 0.3
+       set friction coefficient particles    = 0.3
+       set rolling friction particles        = 0.2
+     end
+     set young modulus wall           = 1e6
+     set poisson ratio wall           = 0.33
+     set restitution coefficient wall = 0.3
+     set friction coefficient wall    = 0.4
+     set rolling friction wall        = 0.2
+   end
+
+
+---------------------------------
+Pneumatic Conveying Simulation
+---------------------------------
+
+The CFD simulation is to be carried out using the slug in the previous step. We will discuss the different parameter file sections.
+The mesh and the DEM boundary condition sections are identical to the ones in the DEM simulations and will not be shown again.
+
+Lagrangian Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The physical properties of the particles are the same as in the DEM simulations, except for the Young's modulus that was increased to use the same value as the article [#lavrinec2021]_.
+
+.. code-block:: text
+
+   subsection lagrangian physical properties
+     set g                        = 0, -9.81, 0
+     set number of particle types = 1
+     subsection particle type 0
+       set size distribution type            = uniform
+       set diameter                          = 0.005
+       set number of particles               = 32194
+       set density particles                 = 890
+       set young modulus particles           = 1e7
+       set poisson ratio particles           = 0.33
+       set restitution coefficient particles = 0.3
```

Comment:
Why is the restitution coefficient so low? isnt that a bit weird? Generally it's around 0.9

Reply:
This is from the paper 
![image](https://github.com/chaos-polymtl/lethe/assets/71884806/3637508e-6c30-490e-9a1a-5d316e17838e)

Reply:
I will look into that

Reply:
They don't say anything about the COR. The material is HDPE.
This paper https://www.sciencedirect.com/science/article/pii/S0009250913005022#t0010 tested polyethylene pellets (they don't say if this is HDPE or LLDPE...) and they get results as low a 0.38. But the particles are not as spherical as the Lavrinec paper say for their particles.
Here they have a COR of 0.5 https://4spepublications.onlinelibrary.wiley.com/doi/epdf/10.1002/pen.20248, many papers that use HDPE cite them.

In my opinion, 0.3 is low, but not as far from 0.5. Honestly, I don't know how they got the value and I do not know if they calibrated the other parameters with it. They do cite a conference abstract about it, but I don't find it anywhere in the interwebs.

Reply:
If its the same value as the paper, then don't worry. Just use the same values that they did. For a flow like this it won't have a huge impact anyway :)

Reply:
I used this value for my simulations, so I really wanted to be sure this was fine :) 
And since I looked at it, I prefer to put some info here than just say "this is from the paper" haha